scripts/index.js

```diff
@@ -0,0 +1,80 @@
+let temp_num = 60;
+
+const change_landscape = (if_temp) => {
+  const earthContainer = document.getElementById("landscape");
+
+  let landscape = "";
+  if (if_temp >= 80) {
+    landscape = "🌵__🐍_🦂_🌵🌵__🐍_🏜_🦂";
+  } else if (if_temp >= 70 && if_temp <= 79) {
+    landscape = "🌸🌿🌼__🌷🌻🌿_☘️🌱_🌻🌷";
+  } else if (if_temp >= 60 && if_temp <= 69) {
+    landscape = "🌾🌾_🍃_🪨__🛤_🌾🌾🌾_🍃";
+  } else if (if_temp <= 59) {
+    landscape = "🌲🌲⛄️🌲⛄️🍂🌲🍁🌲🌲⛄️🍂🌲";
+  }
+  earthContainer.textContent = landscape;
+};
+
+const tempChanges = (temp_num) => {
+  const tempContainer = document.getElementById("temp_num");
+  tempContainer.textContent = temp_num;
+  change_landscape(temp_num);
+};
+
+const increase = () => {
+  temp_num += 1;
+  tempChanges(temp_num);
+};
+
+const decrease = () => {
+  temp_num -= 1;
+  tempChanges(temp_num);
+};
+
+const changeSky = () => {
+  const skySelected = document.getElementById("sky_select").value;
+  const skyOnEarth = document.getElementById("sky");
+  const background = document.body;
+  let sky = "";
+  if (skySelected === "Sunny") {
+    sky = "☁️ ☁️ ☁️ ☀️ ☁️ ☁️";
+  } else if (skySelected === "Cloudy") {
+    sky = "☁️☁️ ☁️ ☁️☁️ ☁️ 🌤 ☁️ ☁️☁️";
+  } else if (skySelected === "Rainy") {
+    sky = "🌧🌈⛈🌧🌧💧⛈🌧🌦🌧💧🌧🌧";
+  } else if (skySelected === "Snowy") {
+    sky = "🌨❄️🌨🌨❄️❄️🌨❄️🌨❄️❄️🌨🌨";
+  }
+  skyOnEarth.textContent = sky;
+  background.className = skySelected;
+};
+
+const state = { location: "" };
+
+const reset = (typed_city) => {
+  //   state.location = "";
+  typed_city = document.querySelector("#typed_city").value;
+  const resetClick = document.querySelector("#typed_city");
+  resetClick.textContent = typed_city;
+};
```

Comment:
you also need to update the city name. I would add a function that gets the header city name and change the input to whatever the user input value is.